src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java

```diff
@@ -80,7 +80,7 @@ public boolean isMoveOver() {
     public void onMovePlayed(int x, int y) {
         int money = arr[x];
         PlayerScores.put(framework.getCurrentPlayer().getName(),
-                PlayerScores.get(framework.getCurrentPlayer().getName()) + money);
+            PlayerScores.get(framework.getCurrentPlayer().getName()) + money);
```

Comment:
Code uses descriptive and meaningful variable, method and class names. Code understanding
does not rely too much on comments.